Documentation on local db and related blocks (#266)
kendraio/kendraio-app: docs/workflow/blocks/load-schema.rst

Load schema  
===========

The Load Schema block is used to load a schema from the "metadata" database and 

> transform it to a jsonSchema which can be used to define a dynamic form. 

Reviewer: ```suggestion
transform it to a JSON schema which can be used to define a dynamic form. 
```

Typical usage
-------------
Used as a sub-block of the "Form" schemaGetter definition. 

.. code-block:: javascript

{
    "type": "form",
    "label": "Save",
    "hasSubmit": false,
    "schemaGetter": {
        "blocks": [
            {
                "type": "load-schema",
                "adapterName": "schemas",

>                 "schemaGetter": "context.schema && context.schema.name"

Reviewer: This example could be simpler too, I think.

            }
        ]
    }
}

Default config
--------------

.. code-block:: javascript

    {
      "type": "load-schema",
      adapterName: 'schemas',
      '//comment': 'use either schema or schemaGetter',
      schema: 'schema',
      schemaGetter: 'data.schema',

>     }

Reviewer: This seems conflicting.
The "default config" example is confusing because there is no block in the "Select Task" GUI that actually produces a block with that config.
How about 2 seperate examples without mentioning what is default?

Supported properties
--------------------
   **adapterName**  - the name of the adapter to use.
    **schema** - the name of the schema to use
    **schemaGetter** - a JMESPath expression to get the schema from the data

   Use either schema or schemaGetter. schemaGetter will override schema.

> Schema data structure (jsonSchema)

Reviewer: ```suggestion
Schema data structure (JSON schema)
```

-----------------------------------

> This block will only work with data conforming to the structure below. 

Reviewer: ```suggestion
This block expects data in a structure like the example below:
```
